openmoc/compatible/opencg_compatible.py

```diff
@@ -753,8 +768,8 @@ def get_openmoc_lattice(opencg_lattice):
   universes = opencg_lattice.universes
 
   # Initialize an empty array for the OpenMOC nested Universes in this Lattice
-  universe_array = np.ndarray(tuple(np.array(dimension[0:2])), \
-                              dtype=openmoc.Universe)
+  # The Lattice universe array is stored x-y-z in OpenCG, but z-y-x in OpenMOC
```

Comment:
You are right that it is indexed by z-y-x in both codes (or at least should be). I made this comment to myself while I was debugging since I had convinced myself otherwise at one point. I'll revert back to the original comment now.